IS0402Test.py

```diff
@@ -816,6 +819,243 @@ def test_30(self):
         else:
             return test.FAIL("Version > 1 not supported yet.")
 
+    def test_31(self):
+        """Query API sends correct websocket event messages for SYNC, ADD, MODIFY and DELETE"""
+
+        test = Test("Query API sends correct websocket event messages for SYNC, ADD, MODIFY and DELETE")
+        api = self.apis[QUERY_API_KEY]
+
+        if self.is04_reg_utils.compare_api_version(api["version"], "v2.0") <= 0:
```

Comment:
...``, "v2.0") <= 0`` means less than *or equal* to v2.0. I think we want ``<`` here.